Samples/iOS-Swift/iOS-Swift/RandomErrors.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+enum SampleError: Error {
+    case bestDeveloper
+    case happyCustomer
+    case awesomeCentaur
+}
+
+class RandomErrorGenerator {
+    
+    static func generate() throws {
+        let random = Int.random(in: 0...2)
```

Comment:
The problem with random is when flaky tests creep in, it's hard to reproduce. So ideally tests should be deterministic

Reply:
I fully agree with you @bruno-garcia, but this is for the sample apps, so I think it is fine.